### DISCUSSION TITLE
BT-805-Lot

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#459] [Q&A] [answered]
Hi Lab
in which notice can I find those field ?
It is defined in fields.json but I cannot find in any subtype notice json.

"id" : "BT-805-Lot",
    "parentNodeId" : "ND-LotGreenCriteria",
    "name" : "Green Procurement Criteria",
    "btId" : "BT-805",
    "xpathAbsolute" : "/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:ProcurementProject/cac:ProcurementAdditionalType[cbc:ProcurementTypeCode/@listName='gpp-criteria']/cbc:ProcurementTypeCode[@listName='gpp-criteria']",
    "xpathRelative" : "cbc:ProcurementTypeCode[@listName='gpp-criteria']",
    "type" : "code",
    "legalType" : "CODE",
    "repeatable" : {
      "value" : false,
      "severity" : "ERROR"
    },
    "forbidden" : {
      "value" : false,
      "severity" : "ERROR",
      "constraints" : [ {
        "noticeTypes" : [ "1", "2", "3", "4", "5", "6", "CEI", "T01", "T02", "X01", "X02" ],
        "value" : true,
        "severity" : "ERROR"
      } ]
    },
    "codeList" : {
      "value" : {
        "id" : "gpp-criteria",
        "type" : "flat"
      },
      "severity" : "ERROR"
    }

In doc : 

Strategic Procurement (BT-06, BT-774, BT-775, BT-776, BT-777, BT-805)
For a given Lot, strategic projects are identified using the "listName" attribute of the "cbc:ProcurementTypeCode" element. The value of the "listName" attribute shall refer to the appropriate codelist ([strategic-procurement](https://op.europa.eu/web/eu-vocabularies/at-dataset/-/resource/dataset/strategic-procurement) for strategic procurement BT-06, [environmental-impact](https://op.europa.eu/web/eu-vocabularies/at-dataset/-/resource/dataset/environmental-impact) for Green Procurement BT-774, [innovative-acquisition](https://op.europa.eu/web/eu-vocabularies/at-dataset/-/resource/dataset/innovative-acquisition) for Procurement of Innovation BT-776, [social-objective](https://op.europa.eu/web/eu-vocabularies/at-dataset/-/resource/dataset/social-objective) for Social Procurement BT-775, gpp-criteria for Green Procurement Criteria BT-805).

Multiple "cac:ProcurementAdditionalType" elements may be used to highlight the different aspects of the procurement (e.g. for an Procurement of Innovation aiming at reducing environmental impacts as illustrated in the example below).

Thanks and Regards
Renato

**Comment by YvesJo** [comment, marked as answer]
BT-805-Lot is a new field that came with the Regulation amendment. Not all fields of the Regulation amendment could be yet added to the Notice Type Definitions and we do our best effort to have them integrated.
